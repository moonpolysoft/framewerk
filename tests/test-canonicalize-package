#! /bin/sh

(
set -e
cd ..
test `fw-exec package/deb/canonicalize-package flass` = flass
test `fw-exec package/deb/canonicalize-package fla~:ss` = fla..ss
set +e
echo " ****** following errors are expected ****** "
fw-exec package/deb/canonicalize-package f && exit 1
fw-exec package/deb/canonicalize-package +f && exit 1
set -e
) > test-canonicalize-package.out 2>&1
