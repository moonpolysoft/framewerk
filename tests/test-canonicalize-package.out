package/deb/canonicalize-package: warning: adjusting nonconforming package 'fla~:ss' -> 'fla..ss'
 ****** following errors are expected ****** 
package/deb/canonicalize-package: fatal: package 'f' needs to be at least length 2.
package/deb/canonicalize-package: fatal: package '+f' does not start with an alphanumeric character.
